src/features/usersActionBar/Sorting.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+import { useAppSelector, useAppDispatch } from '../../app/hooks';
+import {
+  selectFilteredUsers,
+  loadFilteredUsers,
+} from '../users/usersSlice';
+import { SortItem } from '../../types/types';
+
+const Select = styled.select`
+  background: transparent;
+  margin: 0 10px 0 0;
+  outline: none;
+  overflow: hidden;
+`;
+
+const Sorting: FC<SortItem> = ({ columnName }: SortItem) => {
+  const dispatch = useAppDispatch();
+  const filteredUsers = useAppSelector(selectFilteredUsers);
+
+  const deleteEmptyOption = (select: HTMLSelectElement) => {
+    const option = select.querySelector("option[value='']");
+    return option && option.remove();
+  };
+
+  const handleChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    switch (event.target.value) {
+      case 'ASC': {
+        const sortedUsers = filteredUsers.slice().sort((a, b) => {
+          if (a[columnName] > b[columnName]) return 1;
+          if (a[columnName] === b[columnName]) return 0;
+          return -1;
```

Comment:
Здесь, наверное, можно покороче записать:
```suggestion
          return a[columnName] > b[columnName] ? 1 : -1;
```